packages/client/src/views/RecordAudit/ActionDetailsModal.tsx

```diff
@@ -423,24 +423,27 @@ export const ActionDetailsModalListTable = ({
       )}
 
       {/* Correction Requester Id Verified */}
-      {actionDetailsData.requester && (
-        <Table
-          noResultText=" "
-          columns={certificateCollectorVerified}
-          content={[
-            {
-              hasShowedVerifiedDocument:
-                actionDetailsData.hasShowedVerifiedDocument
-                  ? intl.formatMessage(certificateMessages.idCheckVerify)
-                  : intl.formatMessage(certificateMessages.idCheckWithoutVerify)
-            }
-          ]}
-          pageSize={10}
-          totalItems={1}
-          currentPage={currentPage}
-          onPageChange={pageChangeHandler}
-        />
-      )}
+      {actionDetailsData.requester !== 'ANOTHER_AGENT' &&
+        actionDetailsData.requester !== 'REGISTRAR' && (
```

Comment:
```suggestion
      {actionDetailsData.requester !== CorrectorRelationship.ANOTHER_AGENT &&
        actionDetailsData.requester !== CorrectorRelationship.REGISTRAR && (
```
You can import this role from the corrector and use like these
`import { CorrectorRelationship } from '@client/forms/correction/corrector'`

Reply:
okay..would do so..